### PR TITLE
Add examples for mysql and postgres commands

### PR DESCRIPTION
**Pull request by Suna-MS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-rdbms/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.0.5 (2017-06-23)
```
Comment (troydai) [inline src/command_modules/azure-cli-rdbms/HISTORY.rst]: Please change this line to `unreleased`. We will change the version and date when we release packages.

**Comment by troydai** [comment]
@Suna-MS the code changes look fine to me, but I suggest you request someone else from the service team to review the changes. Please also fix the CI.

File: src/command_modules/azure-cli-rdbms/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+unreleased
+++++++++++++++++++
+* Add examples for commands.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-rdbms/HISTORY.rst]: History entries aren't required or desired for help text changes. Only customer facing bug fixes and features.

Comment (Suna-MS) [inline src/command_modules/azure-cli-rdbms/HISTORY.rst]: Will remove the changes in History file

**Pull request merged**